packages/core/stories/patterns/app-header/app-header.stories.tsx

```diff
@@ -0,0 +1,218 @@
+                        onClick={() => setActive(item)}
+                      >
+                        {item}
+                      </NavigationItem>
+                    </li>
+                  ))}
+                  {utilities?.map((utility) => (
+                    <li key={utility.key}>
+                      <NavigationItem orientation="vertical" href="#">
+                        {utility.icon}
+                        {utility.label}
+                      </NavigationItem>
+                    </li>
+                  ))}
+                </ul>
+              </nav>
+            </Drawer>
+          </>
+        ) : (
+          // Desktop Header
+          <FlexLayout
+            style={{
+              height: "calc(var(--salt-size-base) + var(--salt-spacing-200))",
+              paddingLeft: "1em",
+              paddingRight: "1em",
+              backgroundColor: "var(--salt-container-primary-background)",
+              position: "fixed",
+              inset: "0 0 auto 0",
+              borderBottom:
+                "var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-secondary-borderColor)",
+            }}
+            justify="space-between"
+            gap={3}
+          >
+            <FlexItem align="center">App logo</FlexItem>
+            <nav>
+              <ul style={{ display: "flex", listStyle: "none", margin: "0" }}>
+                {items?.map((item) => (
+                  <li key={item}>
+                    <NavigationItem
+                      active={active === item}
+                      href="#"
+                      onClick={() => setActive(item)}
+                    >
+                      {item}
+                    </NavigationItem>
+                  </li>
+                ))}
+              </ul>
+            </nav>
+            <FlexItem align="center">
+              <StackLayout direction="row" gap={1}>
+                {utilities?.map((utility) => (
+                  <Button key={utility.key} variant="secondary">
+                    {utility.icon}
+                  </Button>
+                ))}
+              </StackLayout>
+            </FlexItem>
+          </FlexLayout>
+        )}
+      </header>
+    );
+  };
+
+  const items = ["Home", "About", "Services", "Contact", "Blog"];
+
+  const utilities = [
+    {
+      icon: <SymphonyIcon />,
+      label: "Symphony",
+      key: "Symphony",
+    },
+    {
+      icon: <StackoverflowIcon />,
+      label: "Stack Overflow",
+      key: "Stack Overflow",
+    },
+    {
+      icon: <GithubIcon />,
+      label: "GitHub",
+      key: "GitHub",
+    },
+  ];
+
+  return (
+    <BorderLayout>
+      <BorderItem position="north">
+        <AppHeader items={items} utilities={utilities} />
+      </BorderItem>
+      <BorderItem
+        style={{
+          backgroundColor: "var(--salt-color-purple-30)",
+          padding: "calc(var(--salt-size-base) + var(--salt-spacing-200))",
+          height: "120vh",
```

Comment:
I'm assuming this was done to show the header being sticky? If so we should explain that in the "Main content" area

Reply:
updated